### PR TITLE
Update contribution and frontend readme

### PR DESCRIPTION
**Pull request by BBlackwo**
Changes:
- Add syntax highlighting to code blocks
- Make heading casing consistent (see https://github.com/Start9Labs/embassy-os/pull/1463#discussion_r883889676)
- Add marketplace to frontend readme
- change `build:deps` script to use `npm ci` (to make consistent with the [setup instructions](https://github.com/Start9Labs/embassy-os/blob/master/frontend/README.md?plain=1#L35))

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "check:diagnostic-ui": "tsc --project projects/diagnostic-ui/tsconfig.json --noEmit --skipLibCheck",
     "check:setup-wizard": "tsc --project projects/setup-wizard/tsconfig.json --noEmit --skipLibCheck",
     "check:ui": "tsc --project projects/ui/tsconfig.json --noEmit --skipLibCheck",
-    "build:deps": "rm -rf .angular/cache && cd ../patch-db/client && npm install && npm run build",
+    "build:deps": "rm -rf .angular/cache && cd ../patch-db/client && npm ci && npm run build",
```
Comment (elvece) [inline frontend/package.json:13]: 👍

**Review by elvece — CHANGES_REQUESTED** [review]
Small copy suggestion for accuracy, otherwise looks good, thanks!

File: frontend/README.md [hunk 2]
```diff
@@ -5,7 +5,8 @@ EmbassyOS has three user interfaces and a shared library, all written in Ionic/A
 1. **ui**: the main user interface
 1. **setup-wizard**: used to facilitate initial setup
 1. **diagnostic-ui**: used to display certain diagnostic information in the event EmbassyOS fails to initialize
-1. **shared**: contains components, types, and functions shared amongst the three UIs.
+1. **marketplace**: ui to browse, install, and update packaged services
```
[the remark this answers is not in the document]
Comment (BBlackwo) [reply frontend/README.md]: Updated :+1: Thanks for the better copy!

**Comment by BBlackwo** [comment]
Hi @elvece, I fixed up that marketplace copy if you can have another look. Thanks!

**Review by elvece — APPROVED** [review]

**Pull request merged**